ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -8,7 +9,34 @@ const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM1Savage',
   zoneId: ZoneId.AacLightHeavyweightM1Savage,
   timelineFile: 'r1s.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R1S One-two Paw Right Left',
+      type: 'StartsUsing',
+      netRegex: { id: '9436', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goLeftThenRight(),
+    },
+    {
+      id: 'R1S One-two Paw Left Right',
+      type: 'StartsUsing',
+      netRegex: { id: '9439', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goRightThenLeft(),
+    },
+    {
+      id: 'R1S Biscuit Maker',
+      type: 'StartsUsing',
+      netRegex: { id: '9495', source: 'Black Cat', capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R1S Bloody Scratch',
+      type: 'StartsUsing',
+      netRegex: { id: '9494', source: 'Black Cat', capture: false },
+      response: Responses.aoe(),
```

Comment:
```suggestion
      response: Responses.bigAoe(),
```
This is unsurvivable without mitigation in day 1 gear

Reply:
Added, thanks for the suggestion.